### ISSUE TITLE
tags

### ISSUE THREAD

**Issue by joshp23** [PopVeKind/admin_issue_diagnostic#1]
Great plugin. Could you add tagged releases to the repo so that the [PUNS](https://github.com/joshp23/YOURLS-PUNS) plugin will function properly with it?

**Comment by PopVeKind** [comment]
@joshp23 - Hey Josh!

Sure, if I had an example of the change you wanted.

Your README.md gave no example for plugin developers.
Likewise, your wiki is empty.

So I compared your header to mine.
```
/*
Plugin Name: PUNS - Plugin Update Notification System
Plugin URI: https://github.com/joshp23/YOURLS-PUNS
Description: Provides notification updates for YOURLS plugins under certain conditions
Version: 0.4.4
Author: Josh Panter
Author URI: https://unfettered.net
*/
```
```
/*
Plugin Name: A.I.D. - Admin Issue Diagnostic
Plugin URI: https://github.com/PopVeKind/admin_issue_diagnostic
Description: AID collects environmentals and formats a template to copy/paste into a YOURLS issue. Fantastic for newbies, great for experts!
Version: 1.0
Author: Pop VeKind
Author URI: https://vekind.org/about-vekind/pop-vekind/
TIP: Just fill in the blanks!
*/
```
The only thing I noticed is your `Version:` has three numbers.  Should I change mine to `0.0.1`?

Any suggestions?

**Comment by joshp23** [comment]
Indeed. The PUNS plugin will check against github release tags. So if you look [here](https://github.com/joshp23/YOURLS-PUNS/releases) you can see that there are several releases, each with semantic versioned (the three numbers) names and tags. [Here](https://semver.org/) is some information on semantic versioning, but the PUNS plugin will work with tags the way you have it as well.

**Comment by joshp23** [comment]
I think maybe I was not clear. Just go [here](https://github.com/PopVeKind/admin_issue_diagnostic/releases) in your own repo and draft a new release with the same version tag already present in this plugin's `head`. Following, whenever the tag is updated in the plugin's `head` just make sure to draft a new release with that version number. 

The PUNS plugin will "call home" to this repo as it is listed in the plugin's `head`. It will check this repo's latest release tag, if any, against the installed version's `head` data. I hope that clears it up!

**Comment by joshp23** [comment]
I updated the readme on PUNS for clarity, I hope it helps.
